src/imagelightbox.js

```diff
@@ -453,13 +407,13 @@
             },
 
             _loadPreviousImage = function () {
-                if (options.previousTarget() !== false) {
```

Comment:
right, we had the option (but never added it to the docs) to change default behaviour on previous/next image loads: https://github.com/rejas/imagelightbox/pull/10/ 
but we never added it to the docs and now I am not sure if we need it all / if it works anyway currently :-) so dont be afraid to remove it I'd say, if somebody needs it we can re-add it (with proper docs and exmaples).